config/config.yml

```diff
@@ -221,6 +222,15 @@ arisa:
         - MCPE
```

Comment:
Looks like both modules still only run in MCPE, MCD needs to be added to the projects config here and at KeepPlatform's config.